packages/gatsby-plugin-sharp/README.md

```diff
@@ -67,6 +70,9 @@ Automatically create sizes for different resolutions — we do 1x, 1.5x, and 2x.
 - `width` (int, default: 400)
 - `height` (int)
 - `quality` (int, default: 50)
+- `jpeg_quality` (int)
+- `png_quality` (int)
+- `webp_quality` (int)
```

Comment:
Maybe replace underscores with camelCase as this looks inconsistent now (i.e. `png_quality` but `pngCompressionLevel`)?

Reply:
We also need a note in docs about this feature. I guess the original issue description is a good start for it.